Scripts/script-CommonServer.yml

```diff
@@ -1103,6 +1103,21 @@ script: |-
       email: 'Account.Email(val.Address && val.Address == obj.Address)'
   };
 
+
+  // Converts score (in number format) to human readable reputation format
+  function scoreToReputation(score) {
+      if (score == 3){
+          return 'Bad';
+      }
+      if (score == 2){
+          return 'Suspicious';
+      }
+      if (score == 1){
+          return 'Good';
+      }
+      return 'None';
+  };
+
```

Comment:
I would also remove this and use your new func:

![image](https://user-images.githubusercontent.com/5429033/30103094-44bc8026-92fa-11e7-8cc2-1ddd130cd13c.png)

Reply:
don't understand the remark here.. what to remove?

Reply:
The entire function since you have it in common now

Reply:
that script is deprecated, only for version <=3.1, so can't use that func there